### PR TITLE
[20.01] Fix dependency for CONVERTER_tar_to_directory

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: lib/galaxy/datatypes/converters/tar_to_directory.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <tool id="CONVERTER_tar_to_directory" name="Convert tar to directory" version="1.0.1" profile="17.05">
     <!-- Don't use tar directly so we can verify safety of results - tar -xzf '$input1'; -->
     <requirements>
-        <requirement type="package" version="19.09">galay-util</requirement>
+        <requirement type="package" version="19.9">galay-util</requirement>
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/tar_to_directory.xml]:
```suggestion
        <requirement type="package" version="19.9">galaxy-util</requirement>
```

**Comment by nsoranzo** [comment]
There are other 4 instances of the wrong galaxy-util version introduced in https://github.com/galaxyproject/galaxy/pull/9336

**Review by nsoranzo — APPROVED** [review]

File: lib/galaxy/datatypes/converters/tabular_to_dbnsfp.xml [hunk 2]
```diff
@@ -12,8 +12,8 @@
     </outputs>
     <tests>
         <test>
-            <param name="input" format="tabular" value="gtf_filter_by_attribute_values_list_in3.tabular"/>
-            <output name="dbnsfp" format="snpsiftdbnsfp" value="gtf_filter_by_attribute_values_list_in3.snpsiftdbnsfp"/>
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/tabular_to_dbnsfp.xml:16]: Should gtf_filter_by_attribute_values_list_in3.snpsiftdbnsfp be removed from https://github.com/galaxyproject/galaxy-test-data ?

Comment (mvdbeek) [reply lib/galaxy/datatypes/converters/tabular_to_dbnsfp.xml:16]: yes

Comment (mvdbeek) [reply lib/galaxy/datatypes/converters/tabular_to_dbnsfp.xml:16]: Done in https://github.com/galaxyproject/galaxy-test-data/commit/f82fbd3566dce70e8b1e8a8f43f7c910fb065d10

**Pull request merged**